tests/routes/test_document_families.py

```diff
@@ -4,12 +4,12 @@
 from sqlalchemy.orm import Session
 from fastapi.testclient import TestClient
 from pytest_mock import MockerFixture
-from app.db.models.document.physical_document import (
+from db_client.models.document.physical_document import (
```

Comment:
See Ruff/Black comment

Reply:
Let's have a look together later pelase. We will merge and test this in Staging for now.